README.md

```diff
@@ -1,10 +1,7 @@
 # Composer-enabled Wordpress Upstream template
-## Experimental
 
 This is Pantheon's recommended starting point for forking new upstreams that work with the Platform's integrated
 Composer build process. Replace this with a description of your site. See `upstream-config/README.md` for advice
 on customizing this upstream and instructions on how to use it to create individual sites.
 
-Because it is under active development, you should not create permanent/production sites using this repository
-yet. We make no guarantees of backwards compatibility. Merging new updates from this repository may break existing
-sites.
+For more information and detailed installation guides, please visit the Integrated Composer Pantheon documentation: https://pantheon.io/docs/integrated-composer
```

Comment:
The docs article is not yet published, see https://github.com/pantheon-systems/documentation/pull/5663